More parsers

clementhelsens opened the pull request:
add ```run-final``` and ```run-plots``` subparsers and fix a bug

vvolkl commented on config/FCCAnalysisRun.py:
```diff
@@ -596,7 +596,8 @@ def runStages(args, rdfModule, preprocess):
                 print ('----> Running Locally')
-                runLocal(rdfModule, chunkList[ch], outputchunk, args)
+                args.output = outputchunk
+                runLocal(rdfModule, chunkList[ch], args)
```
Good catch. This could have been caught by a test, still need to add some more ...

vvolkl commented:
Very nice, I would have chosen `plot` and `final` instead of `run-plots` and `run-final`, as shorter is usually better, but looks ok to me

clementhelsens commented:
Thanks @vvolkl , I agree simplier is better and just renamed it. I have a lot of ideas for the overall options we need for the different sub parsers but I might not have the time to implement them by the time of my talk on tuesday. But I think that's fine as the options can be acccessed by ```--help``` for the most up to date ones.
I also changed the running logic https://github.com/HEP-FCC/fcc-tutorials/pull/74/
Do you think we can tag this FCCAnalysis and deploy it soon? same question for the starter kits, can we get them online?
